can/interfaces/vector/canlib.py

```diff
@@ -71,59 +72,64 @@ class VectorBus(BusABC):
     @deprecated_args_alias(**deprecated_args)
     def __init__(
         self,
-        channel,
-        can_filters=None,
-        poll_interval=0.01,
-        receive_own_messages=False,
-        bitrate=None,
-        rx_queue_size=2 ** 14,
-        app_name="CANalyzer",
-        serial=None,
-        fd=False,
-        data_bitrate=None,
-        sjw_abr=2,
-        tseg1_abr=6,
-        tseg2_abr=3,
-        sjw_dbr=2,
-        tseg1_dbr=6,
-        tseg2_dbr=3,
+        channel: Union[int, Sequence[int], str],
+        can_filters: Optional[CanFilters] = None,
+        poll_interval: float = 0.01,
+        receive_own_messages: bool = False,
+        bitrate: int = None,
+        rx_queue_size: int = 2 ** 14,
+        app_name: str = "CANalyzer",
+        serial: int = None,
+        fd: bool = False,
+        data_bitrate: int = None,
+        sjw_abr: int = 2,
+        tseg1_abr: int = 6,
+        tseg2_abr: int = 3,
+        sjw_dbr: int = 2,
+        tseg1_dbr: int = 6,
+        tseg2_dbr: int = 3,
         **kwargs,
-    ):
+    ) -> None:
         """
-        :param list channel:
+        :param channel:
             The channel indexes to create this bus with.
             Can also be a single integer or a comma separated string.
-        :param float poll_interval:
+        :param can_filters:
+            See :class:`can.BusABC`.
+        :param receive_own_messages:
+            See :class:`can.BusABC`.
+        :param poll_interval:
             Poll interval in seconds.
-        :param int bitrate:
+        :param bitrate:
             Bitrate in bits/s.
-        :param int rx_queue_size:
+        :param rx_queue_size:
             Number of messages in receive queue (power of 2).
-            CAN: range 16…32768
-            CAN-FD: range 8192…524288
-        :param str app_name:
-            Name of application in Hardware Config.
-            If set to None, the channel should be a global channel index.
-        :param int serial:
+            CAN: range `16…32768`
+            CAN-FD: range `8192…524288`
+        :param app_name:
+            Name of application in the hardware config.
```

Comment:
```suggestion
            Name of application in Vector Hardware Configuration.
```

This is the name of the Vector tool.